### ISSUE TITLE
[tableio] field MJDdouble for I3EventHeader (Trac #1550)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1542]
Please add a field 'time_start_MJDdouble', as it requires a lot of acrobatics to set the fractured format of day//sec//ns  back together in a proper format.

Migrated from https://code.icecube.wisc.edu/ticket/1550
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:58", 
    "description": "Please add a field 'time_start_MJDdouble', as it requires a lot of acrobatics to set the fractured format of day//sec//ns  back together in a proper format.", 
    "reporter": "mzoll", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067178841456", 
    "component": "combo core", 
    "summary": "[tableio] field MJDdouble for I3EventHeader", 
    "priority": "normal", 
    "keywords": "tableio", 
    "time": "2016-02-16T15:55:56", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160216T16:25:28: cweaver commented:

> I would solve this by extracting `header.start_time.mod_julian_day_double` into an I3Double in the frame immediately prior to writing with tableio. Is there any reason not to prefer this solution; in particular, are there any other fields of the header which tableio wasn't going to write anyway (as part of its necessary row format) which you or other users may want?

**Comment by icecube-trac** [comment]
Trac update at 20160229T17:28:43: 
* olivas changed owner from "" to "jvansanten"
* olivas changed status from "new" to "assigned"
* olivas changed summary from "[tablio] field MJDdouble for I3EventHeader" to "[tableio] field MJDdouble for I3EventHeader"

**Comment by icecube-trac** [comment]
Trac update at 20160328T19:43:58: 
* olivas changed milestone from "offline V16-03-00" to "May 2016"
* olivas changed component from "cmake" to "icecube offline"

**Comment by icecube-trac** [comment]
Trac update at 20160729T09:10:24: 
* jvansanten changed status from "assigned" to "closed"
* jvansanten commented:

> In changeset 148889:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148889"
> Replace fractured but precise representation of event start and end in MJD with
> a double-precision value. This introduces a discretization of 300 ns that will
> increase to 600 ns in the next 100 years and 2500 in the next 1000. These are
> much smaller than the duration of a typical event, and can be ignored in
> coordinate calculations. Fixes #1542.
> ```


* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:58: 
* olivas changed milestone from "May 2016" to ""
* olivas commented:

> Milestone May 2016 deleted
